ui/sse_server.py

```
@@ -0,0 +1,362 @@
+                allow_wildcard = "*" in _ALLOWED_ORIGINS
+                if not allow_wildcard and origin and origin not in _ALLOWED_ORIGINS:
+                    print(f"[DSS SSE] Origin rejected: '{origin}' not in {_ALLOWED_ORIGINS}")
+                    self.send_response(403)
+                    self.send_header("Content-Type", "text/plain")
+                    self.end_headers()
+                    self.wfile.write(b"Forbidden: origin not allowed")
+                    return None
+                if _SSE_TOKEN:
+                    token = self.headers.get("X-DSS-Token", "")
+                    if token != _SSE_TOKEN:
+                        print(f"[DSS SSE] Token rejected")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)